### DISCUSSION TITLE
How to fully disable any coloring - missmatching brackets are always red

### DISCUSSION THREAD

**Discussion by Prinzhorn** [microsoft/monaco-editor#4757] [Q&A]
I'm using `plaintext` to render just vanilla text and noticed the closing mismatching brackets are always red, see https://microsoft.github.io/monaco-editor/playground.html?source=v0.52.0#XQAAAALnAAAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscn80nB77H9Q9THIoLKcy29vgvQRsJtMiV7kY5ycm6kLy_Mes_pZWayLt-K_nKaIWm_27Nk9tWHcxpxKVchZtyoPaStSL3pcRx2JdT-qJNbKKyEYDD7M7ryr_sK76wrnxW_khRFTMX3GO1ENuMNEPPFrbU2tlhi-7muWCUDvOJQGhoC_dP5p4N__7cT2AA

![image](https://github.com/user-attachments/assets/bf9aa27a-16a0-43c3-8bbf-e23d393099ab)

I was not able to find the right option to remove the red color.

**Comment by AncientGrief** [comment]
I have the same problem. Tried `BracketPairColorization.Enabled=false` and `AutoClosingBrackets = "never"` and `MatchBrackets = "never"`

Any solution to this?

**Comment by AncientGrief** [comment]
I did manage to remove it like this:
```css
.monaco-editor .unexpected-closing-bracket {
    color: inherit !important;
}
```

Don't know if this will be working for all use-cases, but for now it seems to work and shouldn't interfer with custom themes.
